scripts/importer/validateData.py

```diff
@@ -727,7 +726,7 @@ def checkSampleId(self, sample_id, column_number):
 
         Return True if the sample id was valid, False otherwise.
         """
-        if sample_id not in DEFINED_SAMPLE_IDS:
+        if DEFINED_SAMPLE_IDS is not None and sample_id not in DEFINED_SAMPLE_IDS:
```

Comment:
I think we should also add a check to verify this is "incremental" upload mode. I.e. the `DEFINED_SAMPLE_IDS` should never be None if the mode is not incremental.

Reply:
Same applies to the other changes below regarding `PATIENTS_WITH_SAMPLES`.

Reply:
> the DEFINED_SAMPLE_IDS should never be None if the mode is not incremental.

It's already taken care of in other place. See here https://github.com/se4bio/cbioportal-core/blob/inc-data-upload-poc/scripts/importer/validateData.py#L5442

Not sure we want to bring extra "incremental" upload mode check into Validator checks.